ui.content/src/main/content/jcr_root/content/forms/af/hclecomm/complaint-form/_jcr_content/guideContainer/rootPanel/items/basics/items/basics1/items/guideimagechoice2/file-1.dir/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : why do we have teh folder named guideimagechoice2 ? and what is its significance ?

Reply:
@rishabh-behl  this default aem name.